docs/pages/graphql-api/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Introduction'
+---
+
+## Liteflow GraphQL API
+
+Liteflow provides a dedicated GraphQL API for your application.
+
+GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data. GraphQL provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
+
+More information about GraphQL on https://graphql.org/
+
+### Playground
+
+GraphQL APIs have an easy learning curve and can be easily testable with a playground that lets you explore and test the API.
+
+[Access the playground](https://nft-test-polygon-mumbai-hdorx.ondigitalocean.app/graphiql)
```

Comment:
Do we really want to already publicly give access to the API without any restriction?

Reply:
They need to have access to a playground to test, this was a great thing to onboard clients. After I agree that it should be on their API. I'm ok to remove this one but for sure, it will be asked for

Reply:
Maybe we can have a sale sentence here: "contact us to get a access to the playground"

Reply:
done